### ISSUE TITLE
Implement Real-Time Face Recognition

### ISSUE THREAD

**Issue by Vibhu-Agarwal** [Marauders-9998/Attendance-Management-using-Face-Recognition#3]
The app currently captures the photo and then train on it or detect faces from it.
A student comes in the class and has to press the capture button to get attendance marked and then the app confirms the student's identity.
The enhancement for the app could be that the student have to just stand in front of the camera, and the app detects the face automatically. Once it recognizes a face, it should then confirm the identity from the student.
Basically, I want to reduce the step of capturing the photo manually by the student.

**Comment by gangubai** [comment]
Hey can you tell me the steps for executing this project? Because I am getting the following things -

ak@Ak-Inspiron-3542:~/A_Projects/Attendance-Management-using-Face-Recognition-master$ python3 Attendance_app.py 
current class Marauders
current class Marauders
ak@Aki-Inspiron-3542:~/A_Projects/Attendance-Management-using-Face-Recognition-master$ python3 capture.py 
ak@Ak-Inspiron-3542:~/A_Projects/Attendance-Management-using-Face-Recognition-master$ python face_train.py 
ak@Ak-Inspiron-3542:~/A_Projects/Attendance-Management-using-Face-Recognition-master$ python3 capture.py 
ak@Ak-Inspiron-3542:~/A_Projects/Attendance-Management-using-Face-Recognition-master$ 

I mean even after adding the images for the students in related folder...please....

**Comment by prakhar154** [comment]
hi, can i work on this issue?

**Comment by Vibhu-Agarwal** [comment]
Hi @prakhar154 
Sure, you can work on this issue. 👍
Since the issue is critical to the application, it would be better if you could raise a thread in the communication channel and have some discussion with the community and maintainers about how you want to solve it.
